modules/project_cleanup/function_source/main.go

```diff
@@ -611,6 +624,33 @@ func invoke(ctx context.Context) {
 		return err
 	}
 
+	removeProjectClusters := func(projectId string) int {
+		logger.Printf("Try to remove clusters for [%s]", projectId)
+		reqLCR := &containerpb.ListClustersRequest{Parent: fmt.Sprintf("projects/%s/locations/*", projectId)}
+		listResponse, err := containerService.ListClusters(ctx, reqLCR)
+		if err != nil {
+			logger.Printf("Failed to list clusters for [%s], error [%s]", projectId, err.Error())
+			return 0
+		}
+
+		logger.Printf("Got [%d] clusters for project [%s]", len(listResponse.Clusters), projectId)
+		if len(listResponse.Clusters) == 0 {
+			return 0
+		}
+
+		for _, cluster := range listResponse.Clusters {
+			logger.Printf("Try to remove cluster: %s", cluster.Name)
+
+			reqDCR := &containerpb.DeleteClusterRequest{Name: fmt.Sprintf("projects/%s/locations/%s/clusters/%s", projectId, cluster.Location, cluster.Name)}
```

Comment:
If a cluster is in stopping or any other state could we still request delete?

https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/projects.locations.clusters#Cluster.Status

Reply:
Yes, good point - Deleting a `STOPPING` seemed to work, however if in `PROVISIONING` (although that status is unlikely during cleanup) it raised an error.  Let me add a switch based on the status as even for STOPPING we can make it more efficient.

Reply:
lgtm!